docs/data-sources/builtin-toolsets/bash.md

```
@@ -95,6 +95,25 @@ Do you want to proceed?
 - **Option 2**: Run and add the prefix to your allow list (saved to `~/.holmes/bash_approved_prefixes.yaml`)
 - **Option 3**: Reject and provide feedback to Holmes
 
+### Importing Claude Code Permissions
+
+If you already maintain allowed Bash commands in Claude Code, import them into Holmes:
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)